src/basic/parse-util.c

```diff
@@ -740,13 +740,13 @@ int parse_oom_score_adjust(const char *s, int *ret) {
 int store_loadavg_fixed_point(unsigned long i, unsigned long f, loadavg_t *ret) {
         assert(ret);
 
-        if (i >= (~0UL << FSHIFT))
+        if (i >= (~0UL << PRECISION_BITS))
```

Comment:
ULONG_MAX is better?

Reply:
I'd rather keep the implementation the same for this PR, to avoid regressions

Reply:
OK.